.changeset/little-pets-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/select': patch
+---
+
+Correct `description` propType
```

Comment:
Can you add what you changed?

Reply:
sorry just seeing this but going to merge as-is to unblock the requestor. I can retroactively make the update in the changelog when I work on select for CCP

Reply:
There's still time! we haven't released as yet